fix: `MethodArgumentSpaceFixer` - fix nested calls for `ensure_fully_multiline` option

geecu opened the pull request:
The inner function call is indented first and then it was ignored when the outer function call was being checked - only the function name god indented, leading to:

```php
foo(
    $x,
    bar(
    $foo,
    $bar,
    $baz
)
);

kubawerlos commented on src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php:
```diff
@@ -150,6 +152,8 @@ public function getPriority(): int
+
         $expectedTokens = [T_LIST, T_FUNCTION, CT::T_USE_LAMBDA, T_FN, T_CLASS];
 
         for ($index = $tokens->count() - 1; $index > 0; --$index) {
```
```suggestion
        $tokenCount = $tokens->count();
        for ($index = 1; $index < $tokenCount; ++$index) {
```
There is no need to introduce `multilinedTokens`, reversing this loop would be enough.

geecu commented:
@kubawerlos your suggestion works like a charm, I've removed `$multilinedTokens` and reversed to order of the loop:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8469/commits/ec747abc7cbdfa6fef75731433fe8827396fd81a

Just in case, I've also added a test with two levels of nested calls to check that this is valid (it is).

kubawerlos commented:
Thank you @geecu
